milv.config.yaml

```diff
@@ -25,3 +25,6 @@ files:
   - path: "./kyma/docs/service-brokers/docs/011-configuration-helm-broker.md"
     config:
       white-list-external: ["custom.bundles-repository"]
+  - path: "./examples/tracing/README.md"
```

Comment:
Are you sure it is the proper path?

Reply:
Thank you for pointing this out - should be in examples.